Formula/rat.rb

```diff
@@ -0,0 +1,26 @@
+class Rat < Formula
+  desc "Compose shell commands to build terminal applications"
+  homepage "https://github.com/ericfreese/rat"
+  url "https://github.com/ericfreese/rat.git"
```

Comment:
This will need to be an actual release for Homebrew to consider it, FWIW.

Reply:
Does it not use `version` anymore?

Reply:
Ok updated:

```diff
diff --git a/Formula/rat.rb b/Formula/rat.rb
index 969d39815c..b286652d46 100644
--- a/Formula/rat.rb
+++ b/Formula/rat.rb
@@ -1,8 +1,7 @@
 class Rat < Formula
   desc "Compose shell commands to build terminal applications"
-  homepage "https://github.com/ericfreese/rat"
+  homepage "https://github.com/ericfreese/rat/archive/v0.0.2.tar.gz"
   url "https://github.com/ericfreese/rat.git"
-  version "0.0.2"
   sha256 "9da8d67a1e16056e3f05cdf536766e5571ede54a6c241c941d6c74eee5d64d31"

   depends_on "go" => :build
```